README.md

```diff
@@ -23,7 +23,7 @@
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
-
+- [@TechnicalShree](https://github.com/TechnicalShree) for his React-Redux example.
 
 ## 📝 License
```

Comment:
Great job @gandradep 🤝

- [ ] [Optional] Kindly fix the MIT license since it appears broken at the moment. This will make your readme file more professional.